common_types/src/datum.rs

```diff
@@ -1274,6 +1274,13 @@ impl<'a> DatumView<'a> {
         }
     }
 
+    pub fn as_time(&self) -> Option<Timestamp> {
```

Comment:
Just name it `timestamp`?
`Time` may be a specific type as I remembered?

Reply:
`as_timestamp` has been used for the variant `Datum::Timestamp`.

Reply:
How about just adding a branch to original `as_timestamp`?